src/ArrowFunctions.h

```diff
@@ -19,6 +19,7 @@ extern "C" {
 #define ARROWUINT32 3
 #define ARROWBOOLEAN 4
 #define ARROWTIMESTAMP ARROWINT64
+#define ARROWSTRING 6
```

Comment:
this prob doesn't matter but should `ARROWSTRING` be 5 since `ARROWTIMESTAMP` points to `ARROWINT64`/0

Reply:
I put ARROWSTRING as 6 because I have another PR in the works for adding float support and that one I reserved for 5, it is really arbitrary though

Reply:
oh okay that makes sense!